research/object_detection/utils/ops_test.py

```diff
@@ -1630,8 +1630,47 @@ def graph_fn():
     self.assertAllClose(expected_gathered_tensor, gathered_tensor_np)
 
 
+class TestGIoU(test_case.TestCase):
 
+  def test_giou(self):
```

Comment:
I think you can expand the tests to be more comprehensive. Examples - a) non overlapping far boxes, b) non overlapping near boxes, c) overlapping boxes where l1/l2 remains same but giou changes with iou